### ISSUE TITLE
Noto Sans Thai:  U+0E1F  and U+0E23 are too similar to 'w' and 's' in Latin. 

### ISSUE THREAD

**Issue by jungshik** [googlei18n/noto-fonts#1049]
### Title 
  The glyph for U+0E1F (ฟ) and U+0E23 (ร) are too similar to 'w' and 's in Latin. 

This is not a bug per se, but there's a complaint at reddit from a Native Thai speaker
( https://www.reddit.com/r/chromeos/comments/7y5lty/need_help_to_change_default_fonts_for_specific/ )

> Hi I'm Chrome OS user from Thailand. Have been using Chrome OS on beta for a year, however in the > latest update (Chrome OS 65 Beta) Google has changed a default font for Thai. Now we're having new > fonts that look prettier especially for those who doesn't read Thai. However this is very hard to read 
> and reduce my productive.

> This new fonts gave me a hard time separate between "s" and "ร" / "w" and "พ" and much more. 

### Font 

Noto Sans Thai
 

### Font Version
  As included in Chrome OS (it's from the latest version as available in github/noto-fonts) 

### OS name and version
  Chrome OS 65

### Issue

  1. Steps to reproduce
       Go to https://goo.gl/SrqUTC on Chrome OS. Alternatively, install Noto Sans Thai and Roboto (or Noto Sans), and type the following in the omnibox on CHrome

```
data:text/html;charset=utf-8,<span style="font-family: Noto Sans Thai, Roboto, Noto Sans;">ฟwรs</span>
```
   
  2. Observed results

![image](https://user-images.githubusercontent.com/8578343/36360133-0a047926-14d6-11e8-9166-3e853118d1f3.png)

![image](https://user-images.githubusercontent.com/8578343/36360225-c8ae1134-14d6-11e8-9348-cecfa0d50840.png)

  3. Expected results
  4. Additional information

**Comment by jungshik** [comment]
@marekjez86 ,  can you go over what feedback collected from Thai native speakers when reviewing a new Thai font (Noto Sans Thai, Noto Sans Thai UI)?

**Comment by ddcarroll** [comment]
Please note that the originator's description of the problem stated "much more" than just the w and s. Most of the characters in the new default Thai font are stylistically different than classical Thai characters. While the font may appear "simpler" for non-Thai viewers, many of the characters are unrecognizable compared to classically written Thai.

**Comment by Incog5** [comment]
I agree that Chrome 65 completely ruined the readability of all Thai fonts across all websites, apps, and input fields:
https://productforums.google.com/forum/#!topic/chromebook-central/fLq7zK4yul8
https://www.reddit.com/r/chromeos/comments/7y5lty/need_help_to_change_default_fonts_for_specific/

**Comment by ddcarroll** [comment]
Good News! Classical Thai font has been restored. My Chromebook just updated to Version 67.0.3396.26 (Official Build) beta (32-bit).

**Comment by Incog5** [comment]
This is very annoying bug/feature.  I'm on Version 66.0.3359.203 (Official Build) (64-bit)
And still waiting on fix.  I concur that the entry box on thai2english is displaying the regular much more readable thai font, while translate.google.com and all other tabs as well as extensions are displaying the cryptic thai font which is very hard to decipher.
Any other places to file bug reports, is it issue with font that chromeos uses, or should the bug be filed with chromeos?

**Comment by jeffmcneill** [comment]
Just FYI, the original complaint is about Pau Paan and "W" not Fau Faan. There are actually four (five, depending how you count) "W" looking characters in Thai:

ผฝพฟฬ

The main thing is that these are differentiated enough from each other. Three have a long right-hand stem (one of those with an additional loop), and the positioning of the "head" aka "loop" and what it looks like is important. 

Note that for the R character ร this transforms into an S-shaped character in many fonts. It shouldn't be an issue as much as the W-shaped characters.

**Comment by thammarith** [comment]
I don't see it becoming a big problem.

Most of Thai loopless typefaces are like this, **พ** is like **w**, **ร** is like **s**, **เ** is like **i**, so on and so forth. For example, one of the most famous Thai loopless fonts, [DB Heavent](http://www.dbfont.biz/dbheavent.php), has the same features.

The edge case that this going to cause confusion is when the users use the Latin scripts with the Thai scripts without spaces, which, in fact, is grammartically wrong. It is an extremely rare case; and the person reading should be knowing that what langauge they are reading.

**Comment by echu888** [comment]
Hi, just wanted to comment on this issue as well.  There really seems to be no way of changing the default font, not through an extension (tried Advanced Font Settings) or anything.  And the loopless Thai font is slightly harder to read.  So, we're completely stuck with the developer's choice of font.  I wouldn't mind either changing the default font out of the box, or allow the user to be able to really set their default font.  Having said that, I'm not sure that ChromeOS actually ships with anything other than Noto Sans Thai / Noto Serif Thai in terms of Thai fonts.  

Why not allow ChromeOS to pull in additional fonts?

**Comment by kimsk132** [comment]
Native Thai speaker here. Coming in to confirm that this font has greatly hinder my productivity, and I much prefer a Thai font that includes the "loop" in the characters.

**Comment by jeffmcneill** [comment]
I also don't understand why there is such a need standardize on sans fonts for scripts where there are usability issues. Someone not doing their homework when it comes to actual legibility and usability. Sad.

**Comment by ddcarroll** [comment]
Homework: Read everything before doing anything.

**Comment by kleindan** [comment]
> Native Thai speaker here. Coming in to confirm that this font has greatly hinder my productivity, and I much prefer a Thai font that includes the "loop" in the characters.

I'm glad to see a native speaker offering their opinion here. As a non-native speaker, I thought I was alone in preferring the "loops" (not sure of the correct term).

**Comment by echu888** [comment]
For what it's worth, this font slows me down [slightly.](url)  But some of my friends have told me that they love the look of the font and have no problems reading it.

**Comment by jeffmcneill** [comment]
Personal preference is important, but it is also useful to look at legibility research on Thai Fonts. Rachapoom has some excellent recent work: https://www.researchgate.net/profile/Rachapoom_Punsongserm

**Comment by kleindan** [comment]
Thank you @jeffmcneill. Per Khun Rachapoom's paper "[Roman-like Thai typefaces: Breakthrough or Regression](https://www.researchgate.net/publication/329335972_Roman-like_Thai_typefaces_Breakthrough_or_Regression)?", emphasis mine:

_However, the progressive ideology of Thai designers means that it is always intentionally used while **reducing the readability of information**. In addition, **Thai people did not recognize Roman-like Thai typefaces as easily** as conventional Thai typefaces._

**Comment by simoncozens** [comment]
I'm going to close this; Noto has for better or worse gone with this kind of somewhat Latinized(?) design for Noto Sans Thai, and a complete redesign of the font with the same name is not on the cards at this point. There is a Noto Looped Thai for the "traditional" style where that is preferred.
